spec/facets/ColumnLineageDatasetFacet.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://openlineage.io/spec/facets/1-0-2/ColumnLineageDatasetFacet.json",
+  "$id": "https://openlineage.io/spec/facets/1-0-3/ColumnLineageDatasetFacet.json",
```

Comment:
I would bump this to 1.1.0 or even 2.0.0, it's pretty large change :)

Reply:
done